tasks/packages.yml

```diff
@@ -17,8 +17,30 @@
 # limitations under the License.
 #
 
+- include: packages_ubuntu.yml
+  when: ansible_distribution == 'Ubuntu'
```

Comment:
I like keeping `ansible_os_family` for both, you may want to install Openstack in a ubuntu derivative for example.
Also, service names for handlers depend on them.

BTW, there are Debian packages for openstack (a little bit outdated and undocumented though). I don't think the role will work on it, but just I'm just pointing the fact